<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1918</width>
    <height>1109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>631</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>current image</string>
    </property>
    <widget class="QLabel" name="labelImg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>612</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxGImg">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>1241</width>
      <height>1061</height>
     </rect>
    </property>
    <property name="title">
     <string>global image</string>
    </property>
    <widget class="QLabel" name="labelGImg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1224</width>
       <height>1024</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>functions</string>
    </property>
    <widget class="QPushButton" name="pushButtonScan">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonInit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>init</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1918</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
